apps/autoscheduler/Makefile

```diff
@@ -63,12 +66,6 @@ $(BIN)/demo.a: $(BIN)/demo.generator $(BIN)/auto_schedule.so
 $(BIN)/demo.rungen: $(BIN)/demo.a $(BIN)/runtime.a
 	$(CXX) -std=c++11 -I $(BIN) -DHL_RUNGEN_FILTER_HEADER="\"demo.h\"" -I ../../include ../../tools/RunGenMain.cpp ../../tools/RunGenStubs.cpp  $(BIN)/demo.a -o $@ $(HALIDE_SYSTEM_LIBS) -lpng -ljpeg
 
-$(BIN)/test: test.cpp $(BIN)/auto_schedule.so
```

Comment:
Oops, I forgot to check in test.cpp originally. I've added it, so please restore this block of the makefile.